### ISSUE TITLE
Allow ExtractOIIOTranscode without colorspace, to just scale

### ISSUE THREAD

**Issue by alainxi** [ynput/ayon-core#1003]
### Is there an existing issue for this?

- [x] I have searched the existing issues.

### Please describe the feature you have in mind and explain what the current shortcomings are?

Currently, if you want to use OIIO, you need to enable Color Management in your DCC, even if you don't want to use colorspace.
It would be useful to allow ExtractOIIOTranscode to support a setting like "Dont convert colorspace" so that we can also use it to just convert filetype or perform scaling, even if NO colorspace data exists in the input instance.
Because in those cases we don't want to mess with the colorspace of the image but JUST want to resize.

### How would you imagine the implementation of the feature?

I would like to avoid enabling `Color Management` in DCCs, when I just want to scale textures.

### Are there any labels you wish to add?

- [x] I have added the relevant labels to the enhancement request.

### Additional context:

Discussed with Mustafa and Roy, here :

https://community.ynput.io/t/publish-from-substance-painter-output-several-resolutions/1971/3

**Comment by alainxi** [comment]
Another consequence of OIIO being forced to use color :  Ayon code only allows to use OIIO if the input maps have RGB channels.
So, you can't resize a single-channel map.
For example it's a problem for textures "Roughness", "Metallic" and "Height", that are exported by Substance Painter with the "PBR Metallic Roughness' Output Template.

Discussed here :
https://discord.com/channels/517362899170230292/563751989075378201/1309524464685420616

**Comment by MustafaJafar** [comment]
I wonder if it can be done the other way? having two exatractors one is general purpose and the other is for color transcode.

**Comment by alainxi** [comment]
Yes, it would provide a great freedom for general purpose extractor (allowing single-channel maps, resize, etc...), without risking to break stuff on the current transcode.

**Comment by BigRoy** [comment]
I personally don't think we need a separate extractor for that. We should just make an OIIO extractor can do most of what you'd need to with `oiiotool` - including conversions without colorspace conversions.

**Comment by MustafaJafar** [comment]
> I personally don't think we need a separate extractor for that. We should just make an OIIO extractor can do most of what you'd need to with `oiiotool` - including conversions without colorspace conversions.

It be great. from user experience perspective, I wonder if it'd lead to confusion. so, how can we reflect that on the plugin label and setting UI.

**Comment by alainxi** [comment]
This topic is also discussed on `Ayon Feedback` :

https://feedback.ayon.app/en/p/allow-extractoiiotranscode-without-colorspace-to-just-scale
